mne_bids/write.py

```diff
@@ -819,6 +820,11 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
         inferred from the stim channel using `mne.find_events`.
     event_id : dict | None
         The event id dict used to create a 'trial_type' column in events.tsv
+    anonymize : bool | dict
```

Comment:
If `anonymize` is True but `deface` is `False` I think you should throw an error. Otherwise, people might mistakenly assume that defacing has been done.

Reply:
I think deface is only for write_anat and anonymize is only for write_raw_bids